src/language_features/document_symbol.rs

```diff
@@ -23,6 +23,125 @@ use std::path::{Path, PathBuf};
 use unicode_width::UnicodeWidthStr;
 use url::Url;
 
+pub fn breadcrumbs(meta: EditorMeta, editor_params: EditorParams, ctx: &mut Context) {
+    let eligible_servers: Vec<_> = ctx
+        .language_servers
+        .iter()
+        .filter(|srv| attempt_server_capability(*srv, &meta, CAPABILITY_DOCUMENT_SYMBOL))
```

Comment:
unrelated: one of these days we should go through all the requests that are not yet gated on a capability.
Otherwise we might get errors, especially from "mixin" servers that support only a handful of things like diagnostics